dashboard/css/style.css

```diff
@@ -246,3 +246,7 @@ nav.minimize ~ .dashboard {
     opacity: 1;
   }
 }
+
+.testCss {
+  background-color: #181818;

```

Comment:
@ModitTdr  Please change the background color to white

Reply:
done